eq-author-api/schema/tests/duplication.test.js

```diff
@@ -0,0 +1,158 @@
+const { last, omit } = require("lodash");
+
+const {
+  buildQuestionnaire,
+} = require("../../tests/utils/questionnaireBuilder");
+
+const {
+  queryQuestionnaire,
+  deleteQuestionnaire,
+  duplicateQuestionnaire,
+} = require("../../tests/utils/questionnaireBuilder/questionnaire");
+
+const {
+  querySection,
+  deleteSection,
+  duplicateSection,
+} = require("../../tests/utils/questionnaireBuilder/section");
+
+const {
+  queryQuestionPage,
+  deleteQuestionPage,
+  duplicatePage,
+} = require("../../tests/utils/questionnaireBuilder/page");
+
+describe("Duplication", () => {
+  let questionnaire, section;
+  let config = {
+    sections: [
+      {
+        title: "section-title-1",
+        alias: "section-alias-alias-1",
+        pages: [
+          {
+            title: "page-title-1",
+            alias: "page-alias-alias-1",
+          },
+        ],
+      },
+    ],
+  };
+
+  beforeAll(async () => {
+    questionnaire = await buildQuestionnaire(config);
+    section = last(questionnaire.sections);
+  });
+
+  afterAll(async () => {
+    await deleteQuestionnaire(questionnaire.id);
+  });
+
+  describe("duplicate a page", async () => {
+    let page, pageCopy;
+
+    beforeEach(async () => {
+      page = await queryQuestionPage(questionnaire, last(section.pages).id);
+      let { id } = await duplicatePage(questionnaire, page);
+      pageCopy = await queryQuestionPage(questionnaire, id);
+    });
+
+    afterEach(async () => {
+      await deleteQuestionPage(questionnaire, pageCopy.id);
```

Comment:
I'm not sure I like sharing questionnaires across tests. I think we should discuss this after. My worry is that we could have state leaking between tests